packages/ui-color-picker/src/ColorMixer/ColorPalette/index.tsx

```diff
@@ -234,8 +238,11 @@ class ColorPalette extends Component<ColorPaletteProps, ColorPaletteState> {
         tabIndex={this.props.disabled ? undefined : 0}
         onKeyDown={(e) => this.handleKeyDown(e)}
         onMouseDown={(e) => this.handlePaletteMouseDown(e)}
-        aria-label={this.props.navigationExplanationScreenReaderLabel}
+        id={this._id}
       >
+        <ScreenReaderContent>
+          {this.props.navigationExplanationScreenReaderLabel}
+        </ScreenReaderContent>
```

Comment:
So this is the part we have to redo: remove the `ScreenReaderContent` and put the `aria-label` back. 

The tests will throw the error again, so we agreed with the a11y team to ignore these errors. This is the way to do it:

```js
// in `generateA11yTests`
generateA11yTests(Component, ComponentExamples, [
  'name-of-the-aria-error'
])

// in `accessible`
expect(
    await component.accessible({ ignores: ['name-of-the-aria-error'] })
  ).to.be.true()
})
```